app/presenters/track.rb

```diff
@@ -47,12 +47,12 @@ def topic_content(topic_key)
       end
 
       def contributing_instructions(topic_key)
-        template = File.read("./x/docs/md/track/BETTER.md")
+        template = File.read(shared_template_filepath("BETTER"))
         populate_template_with_trackler_data(template).gsub('TOPIC', topic_key.to_s.upcase)
       end
 
       def fallback_topic_content(topic_key)
```

Comment:
`def contributing_instructions(topic_key)`
and 
`def fallback_topic_content(topic_key)`

Look awfully similar, I wonder if keyword arguments could help us here by making explicit what topic_key is going to be used for.

`def contributing_instructions(replacement_value: topic_key)`
and 
`def fallback_topic_content(file_basename: topic_key)`